ChangeLog.md

```diff
@@ -1,6 +1,18 @@
 
 # Change Log
 
+## Version 10.30.1
+
+### Bug Fixes 
+1. Fixed `--exclude-path` flag not available in remove operations.([#3165](https://github.com/Azure/azure-storage-azcopy/pull/3165))
+2. Fixed regression where AzCopy was not honoring concurrency value in copy operations ([#3192](https://github.com/Azure/azure-storage-azcopy/pull/3192))
+3. Fixed the incorrect JSON output format of the warning message when there are multiple AzCopy processes running. ([#3188](https://github.com/Azure/azure-storage-azcopy/pull/3188))
+4. Fixed `latest_version.txt` from being wrongly created in users current directory. ([#3179](https://github.com/Azure/azure-storage-azcopy/pull/3179))
+5. Fixed AzCopy crashing during sync operation from a nil pointer deref in the destination authentication policy. ([#3186](https://github.com/Azure/azure-storage-azcopy/pull/3186))

```

Comment:
I think there are a few Github issues these bug fixes are related to - could you link them as well?